lib/matplotlib/tests/test_backend_ps.py

```diff
@@ -71,7 +71,13 @@ def test_savefig_to_stringio(format, use_log, rcParams, orientation):
 
         assert not s_buf.closed
         assert not b_buf.closed
-        s_val = s_buf.getvalue().encode('ascii')
+        try:
+            s_val = s_buf.getvalue().encode('ascii')
+        except UnicodeEncodeError:
+            print('string:',
+                  s_buf.getvalue().encode('ascii', errors='backslashreplace'))
+            print('bytes:', b_buf.getvalue())
+            raise
```

Comment:
Can we move this to a separate PR? These are two independent issues and we're not yet clear whether this here is the correct solution #22416. So it's better address it separately.

Pinning coverage reduces the number of failed CI jobs and should go in as soon as possible.

Reply:
This is not a solution, just some debug testing. I've removed it.